package.json

```diff
@@ -163,15 +160,21 @@
     "eslint": "^8.49.0",
     "eslint-plugin-absolute-imports": "^0.0.3",
     "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "graphql-cli": "4.1.0",
+    "eslint-plugin-typescript-sort-keys": "^2.3.0",
+    "jest": "^29.2.1",
+    "jest-expo": "~50.0.1",
+    "jest-transform-stub": "^2.0.0",
     "openapi-typescript-codegen": "^0.23.0",
     "patch-package": "^6.5.1",
     "postinstall-postinstall": "^2.1.0",
     "prettier": "^2.7.1",
     "prettier-plugin-organize-imports": "^3.2.2",
     "react-native-svg-transformer": "^1.3.0",
+    "react-test-renderer": "^18.2.0",
     "timezone-mock": "^1.3.6",
+    "ts-node": "^10.9.2",
```

Comment:
Unclear why this is necessary, perhaps as a side-effect of some dependency being added? Without it, I see:

```
$ jest
Error: Jest: Failed to parse the TypeScript config file /home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/jest.config.ts
  Error: Jest: 'ts-node' is required for the TypeScript configuration files. Make sure it is installed
Error: Cannot find package 'ts-node' imported from /home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/readConfigFileAndSetRootDir.js
    at readConfigFileAndSetRootDir (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/readConfigFileAndSetRootDir.js:116:13)
    at async readInitialOptions (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/index.js:403:13)
    at async readConfig (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/index.js:147:48)
    at async readConfigs (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/index.js:424:26)
    at async runCLI (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/@jest/core/build/cli/index.js:151:59)
    at async Object.run (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-cli/build/run.js:130:37)
error Command failed with exit code 1.
```

Reply:
Ok, so this is what resolved the `jest` failure related to not being able to find the `expo/tsconfig.base` file. My solution was to configure our `tsconfig.json` with the same values.

Hosting `ts-node` into our dependencies looks like it allows `ts-node` to have access to the `expo` module.